influxql/functions.go

```diff
@@ -355,11 +347,19 @@ type FloatHoltWintersReducer struct {
 }
 
 const (
-	defaultAlpha   = 0.5
-	defaultBeta    = 0.5
-	defaultGamma   = 0.5
-	defaultPhi     = 0.5
+	// Arbitrary weight for initializing some intial guesses.
+	// This should be in the  range [0,1]
+	weight = 0.5
```

Comment:
I'd rather not start prefixing a bunch of vars. Should we move this code into its own package? Its just implementing an interface so its should be easy to move.